dpppt-backend-sdk/dpppt-backend-sdk-data/src/main/java/org/dpppt/backend/sdk/data/gaen/JDBCGAENDataServiceImpl.java

```diff
@@ -127,6 +132,11 @@ public int getMaxExposedIdForKeyDate(
             + " and rolling_start_number < :rollingPeriodStartNumberEnd and received_at <"
             + " :publishedUntil";
 
+    params.addValue(
+        "maxAllowedStartNumber",
+        now.roundToPreviousBucket(releaseBucketDuration).plusHours(2).get10MinutesSince1970());
```

Comment:
@UBaggeler : discussing this with @ubamrein and @martinalig rang a bell about a special treatment for iOS, but nobody remembered exactly why.

The most logical thing to us would be to write:
```suggestion
        now.roundToPreviousBucket(releaseBucketDuration).get10MinutesSince1970());
```

Does that sound OK to you?